config_example.sh

```diff
@@ -74,3 +74,8 @@
 # Number of maximum verification or test retries
 #
 #export TEST_MAX_TIME=120
+
+#
+# Set the driver
+#
+#export BMC_DRIVER="redfish"
```

Comment:
we should maybe add a note that we use ipmi by default. or switch to redfish by default ?

Reply:
Done